### PR TITLE
fix: add twine to upload to PyPi

### PR DESCRIPTION
**Pull request by abtris**
This can solve #161 

Used steps from https://pypi.org/project/twine/

### CODE REVIEW

File: pavement.py [hunk 1]
```diff
@@ -119,5 +119,5 @@ def release():
     sh("git tag -s '%(name)s-%(version)s' -m 'Version bump to %(version)s'" % {'name': NAME, 'version': VERSION})
     sh("git push --tags")
     sh("git push")
-    sh("python setup.py register sdist upload")
```
Comment (honzajavorek) [inline pavement.py]: I’m not completely sure whether we should remove `register`. Is it deprecated as well?

Comment (abtris) [reply pavement.py]: I'm not sure but works without that. But we have already register. I will get this back. 

https://docs.python.org/3.1/distutils/packageindex.html

Comment (abtris) [reply pavement.py]: Fixed, I added `register` back.

Comment (honzajavorek) [reply pavement.py]: It's true the `register` is only needed once...

File: pavement.py [hunk 2]
```diff
@@ -119,5 +119,5 @@ def release():
...
-    sh("python setup.py bdist_wheel upload")
+    sh("python setup.py register sdist bdist_wheel")
```
Comment (kylef) [inline pavement.py]: @honzajavorek is right and we don't need register. I'm doing [similar](https://github.com/kylef/maintain/blob/master/maintain/release/python.py#L38-L41):

```python
        version = self.determine_current_version()
        invoke(['python', 'setup.py', 'sdist', 'bdist_wheel'])
        invoke(['twine', 'upload', 'dist/*{}*'.format(version)])
```

File: pavement.py [hunk 3]
```diff
@@ -119,5 +119,5 @@ def release():
...
+    sh("twine upload dist/*")
```
Comment (kylef) [inline pavement.py]: Maybe a better idea to make this `dist/*{version}*` so we don't re-upload old versions of mistakenly upload wrong versions.

**Comment by abtris** [comment]
@kylef updated

**Comment by abtris** [comment]
Pls, re-review.

File: pavement.py (hunk 2 above)
Comment (honzajavorek) [reply pavement.py]: I wasn't right, because first I made @abtris to put `register` back 😄

**Pull request merged**